src/_assets/css/_clock.scss

```diff
@@ -0,0 +1,214 @@
+.clock {
+  $vertical-line-color: #384753;
+  
+  background-color: $flutter-dark-blue-texture;
+  font-family: $site-font-family-alt;
+
+  .center {
+    position: absolute;
+    top: 53%;
+    transform: translateY(-50%);
+    width: 100%;
+    z-index: 99;
+  }
+
+  .heading {
+    animation: slide-in 1s .3s;
+    animation-fill-mode: both;
+    color: $site-color-white;
+    font-size: 18px;
+    margin: 0 0 24px 0;
```

Comment:
You can omit the last cero. `margin: 0 0 24px;`

Reply:
Done